docs/adhara/crosschain_protocols.md

```diff
@@ -0,0 +1,71 @@
+
+In a public network, that relies on a proof of work (PoW) consensus mechanism, any (mining) node on the network can propose the next block of transactions, and every node has to verify and execute the block of transactions on their local copy of the network. In such a system very little trust is required between nodes, but it comes at the expense of high latency and only achieving probabilistic finality of blocks, at best.
+
+In a permissioned network, that relies on a proof of authority (PoA) consensus mechanism (e.g. QBFT), a chosen set of trusted authorities or validators get to propose the next block of transactions. It is, however, still up to each node to verify and execute the block of transactions before adding it to their local copy of the network. In such a system more trust is given to the authorities or validators, but nodes still have the option to verify each block and transaction. It results in a drastic reduction in latency and immediate finality of blocks. Validators sign each block they have verified, and validity consensus is reached between validators on the network.
+
+In a network where pairs of actors have a bilateral communication channel, it could simply be two actors and a notary who confirms that a transaction (e.g. state update) took place. In such a system the two actors would need to have a high level of trust in the notary, but there is even further improvements to latency, transaction throughput and privacy.
+
+Taking it a step further, what would the level of trust between two (potentially) different DLT networks look like? As with a single distributed system, where there are tradeoffs between decentralised (i.e. trustless) and centralised (i.e. trusted) authority, there are also tradeoffs between different interoperability protocols use by distributed systems. On the one end, we could program each system to interpret and keep state of the other system, which would be extremely complex and hard to maintain. On the other end, we could trust an intermediary, creating a single point of failure.  
+
+As we will explore below, one can find a middle ground, where an external system can be programmed to interpret certain types of events or transactions they care about. This protocol does not rely on an intermediary to confirm that a block or transaction does what it needs to, but instead rely on one or more (already) trusted parties, on the network where the event or transaction happened, to deem the block or transaction valid.
+
+## Ethereum block header proofs 
+
+There are some standard patterns that can assist with verifying that a transaction or event on an Enterprise Ethereum network is in fact valid. This pattern is repeatable for all types of transactions or events, and is therefor scalable because it doesn't rely on knowledge about the actors involved in the transaction or event, nor the rules of the network, only on the validators of that network.
```

Comment:
```suggestion
There are some standard patterns that can assist with verifying that a transaction or event on an Enterprise Ethereum network is in fact valid. This pattern is repeatable for all types of transactions or events, and is therefore scalable because it doesn't rely on knowledge about the actors involved in the transaction or event, nor the rules of the network, only on the validators of that network.
```